Logo updates

strixaluco opened the pull request.

vyp commented:
This is really awesome! :smiley:

@nylira is it okay for some of the rejected logos to be 64x64px instead of 32x32px as before? Or should they be scaled down?

vyp commented:
Is this PR a work in progress? Unless I missed them, I'm not seeing the logo changes that #1774 would introduce. Namely `facebook-messenger.png`, `apple-macos.png`, `google-analytics.png`, `whatsapp.png`, `tumblr.png`.

vyp commented:
Woops, spoke too soon, `whatsapp.png` already exists.

strugee commented on source/db/zh-TW-projects.json:
```diff
@@ -1135,7 +1135,7 @@
     "description": "POSIX-compliant enterprise cryptographic stacked filesystem for GNU/Linux.",
     "license_url": "https://git.kernel.org/cgit/linux/kernel/git/tyhicks/ecryptfs.git/tree/COPYING",
-    "logo": "geli.png",
+    "logo": "encryptfs.png",
```
Would be better to simply remove all these duplicated translation keys and let them inherit from the English JSON, instead of changing all of them. I'd be okay merging as-is though.

strugee commented:
@vyp I'd be uncomfortable trying to coordinate changes across multiple PRs. AFAICT this doesn't rename anything right? And it's clearly an improvement. So let's review and merge this, and then someone can rebase #1774.

strugee commented:
Whoever merges this should also use "squash and merge" and fix the commit message so the subject line is decently short and the list of logos is in the commit message body.

strixaluco commented:
@strugee 
Fixed `ecryptfs` typo and added it only to `en-projects.json`. Didn't know that other languages are generated from it.
Also squashed everything in one commit.

vyp commented:
> @vyp I'd be uncomfortable trying to coordinate changes across multiple PRs. AFAICT this doesn't rename anything right? And it's clearly an improvement. So let's review and merge this, and then someone can rebase #1774.

Yep good idea I think that would be easier.

strixaluco commented:
So should I combine #1776 with #1774?

vyp commented:
Don't think there's a need to do that. Just waiting on nylira for now to merge this.

strixaluco commented:
Fixed everything mentioned in your second post, @vyp

vyp commented:
Sorry to be a bother, but maybe these latest changes (where you added the images I mentioned in the second comment), should be part of #1774 instead. That way the PRs are not inter-related and can be merged separately without it breaking the build. But if you don't have the time don't worry and keep as is, both of these PRs will be merged around the same time then, so no big deal.

strixaluco commented:
Okay, will try to separate them tonight.

vyp commented:
@strixaluco Just an update, I will try to review and merge this and #1774 on Thursday. :wink:

strixaluco commented:
@vyp Thanks for the update. I'm sorry for not fixing this, maybe should've started from scratch to make it easier.

vyp commented:
@strixaluco No sweat :smile: 

Unless it's just me, there seems to be something wrong with google-analytics.png, can you confirm?

strixaluco commented:
@vyp Yeah, google-analytics.png seemed to be a black pixel reviewing PR here at GitHub, so I decided to fix it along with scaling icons down to 32×32 px, but that didn't help, unfortunately. It's still a black pixel.
Could you please check google-analytics.png locally? I haven't had any issue with it.

vyp commented:
Yep, it looks good locally. :sparkles:

strixaluco commented:
Cool! Not at all :) Thanks for reviewing and merging.
